packages/transport-test/e2e/bridge/controller.ts

```
@@ -11,6 +11,8 @@ export const env = {
     USE_NODE_BRIDGE: process.env.USE_NODE_BRIDGE === 'true',
 };
 
+type TrezordNode = any;
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TrezordNode'. Consider to delete it or rename it.

'TrezordNode' is defined here:


(lint/suspicious/noRedeclare)